java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1404,9 +2009,6 @@ public final Builder setNullAt(long index) {
      * Finish and create the immutable CudfColumn.
      */
     public final HostColumnVector build() {
-      if (built) {
-        throw new IllegalStateException("Cannot reuse a builder.");
-      }
```

Comment:
Do we want to reuse a builder now?

Reply:
I have for now made it so that u cannot. Maybe we will in the future..